bugsnag-android-core/src/main/java/com/bugsnag/android/DeliveryDelegate.java

```diff
@@ -45,7 +45,10 @@ void deliver(@NonNull Event event) {
             }
         }
 
-        if (event.isUnhandled()) {
+        boolean unhandledOverridden = event.impl.getUnhandledOverridden();
+        boolean unhandled = event.isUnhandled() || unhandledOverridden;
```

Comment:
If the user has changed a handled event to unhandled, it should not be treated as unhandled.

Perhaps rather than expose `getUnhandledOverridden`, we should expose the original handled value so it can be easily queried here?

Reply:
Added an internally visible method named `getOriginalUnhandled()` which is now used here.